### PR TITLE
Assigning static IPs works only for some SDN plug-ins

### PR DESCRIPTION
**Pull request by jboxman**
- Can use only ovs-networkpolicy or ovs-multitenant
- https://bugzilla.redhat.com/show_bug.cgi?id=1702361

### CODE REVIEW

**Comment by jboxman** [comment]
Hi @rbbratta,

Can you verify if this docs update this correct? (I pulled your name from the _Team Member Tracking AOS_ Google sheet, which lists you as a primary for _openshift-sdn_.)

Thanks!

File: admin_guide/managing_networking.adoc [hunk 1]
```diff
@@ -1184,6 +1184,11 @@ resources can recognize the traffic.
 Unlike the egress router, this is subject to `EgressNetworkPolicy` firewall
 rules.
 
+[NOTE]
+====
+Assigning static IPs addresses for projects in your cluster requires the SDN to use either the *ovs-networkpolicy* or *ovs-multitenant* network plug-ins.
```
Comment (kalexand-rh) [inline admin_guide/managing_networking.adoc]: The package names need to be in [backticks](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference).

Also, where do we talk about these packages? I don't either of those names in the master branch.

**Review by kalexand-rh — COMMENTED** [review]
LGTM

**Comment by jboxman** [comment]
@kalexand-rh, thanks!

**Comment by jboxman** [comment]
Verified on the bug; I'm going to merge this.

**Pull request merged**